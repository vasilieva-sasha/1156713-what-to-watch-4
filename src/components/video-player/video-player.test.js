import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

const mock = {
  id: 1,
  poster: ``,
  preview: ``
};

describe(`VideoPlayer`, () => {
  it(`Video player renders correctly`, () => {
    const tree = renderer.create(
        <VideoPlayer isPlaying={false} film={mock} />
    );

    expect(tree).toMatchSnapshot();
  });
});
