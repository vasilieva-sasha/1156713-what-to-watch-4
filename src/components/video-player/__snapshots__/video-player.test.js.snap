// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer Video player renders correctly 1`] = `
<video
  poster="img/.jpg"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
